MonstieWash/Assets/Scripts/Erasing/Eraser.cs

```diff
@@ -116,8 +128,21 @@ public void UseTool()
             }
         }
 
-        if (!wasErasing && IsErasing) OnErasing_Started?.Invoke();
-        if (wasErasing && !IsErasing) OnErasing_Ended?.Invoke();
+        if (!wasErasing && IsErasing) OnErasing_Started?.Invoke(false);
+        if (wasErasing && !IsErasing) OnErasing_Ended?.Invoke(false);
+    }
+
+    public void UseToolClean()
+    {
+        if (!IsErasingClean && (distFromCentre < 3f && m_taskTracker.IsThisSceneComplete()))
+        {
+            OnErasing_Started?.Invoke(true);
+            IsErasingClean = true;
+        }
+        if (IsErasingClean && distFromCentre > 3f)
+        {
+            StopUseToolClean();
+        }
```

Comment:
Sure. I had it set to a magic number because it fit the size of the mimic/slime and technically can't be changed on a monster-by-monster basis anyway (since it's attached to the eraser, not the monster AI). But will make the change